### PR TITLE
Support haml, slim, coffeescript in generators

### PR DESCRIPTION
**Pull request by nekath**
Hot reloader generator should check template language of application layout. 
Added support for Slim and Haml.

Core generator should add requires for needed bundled files (react_bundle and react_integration) in application.js or application.coffee.

Bump rails versions in test apps

### CODE REVIEW

File: lib/generators/react_webpack_rails/install/core_generator.rb [hunk 1]
```diff
@@ -42,6 +43,31 @@ def webpack
...
+        case File.extname(manifest_file)
+        when '.coffee'
+          requires = <<-REQUIRE_STRING.strip_heredoc
+            #= require react_integration
+            #= require react_bundle
+          REQUIRE_STRING
+
+          inject_into_file manifest_file, requires, before: '#= require_tree .'
+          append_to_file manifest_file, requires
+        else
```
Comment (mxaly) [inline lib/generators/react_webpack_rails/install/core_generator.rb]:
```
else if '.js' 
```
and inform about not matching extension.

File: lib/generators/react_webpack_rails/install/core_generator.rb [hunk 2]
```diff
@@ -42,6 +43,31 @@ def webpack
         copy_file 'webpack/dev.config.js', 'webpack/dev.config.js'
         copy_file 'webpack/production.config.js', 'webpack/production.config.js'
       end
+
+      private
+
+      def require_bundles_in_application_js
+        manifest_file = Dir.glob("app/assets/javascripts/application.*").first
+
+        case File.extname(manifest_file)
```
Comment (mxaly) [inline lib/generators/react_webpack_rails/install/core_generator.rb]: what if there will be `.js.erb` or `.coffee.erb` ?

File: lib/generators/react_webpack_rails/install/core_generator.rb [hunk 3]
```diff
@@ -42,6 +43,31 @@ def webpack
...
+        when '.coffee'
```
Comment (mxaly) [inline lib/generators/react_webpack_rails/install/core_generator.rb]: \.coffee(.erb)?/

**Comment by nekath** [comment]
@mxaly Fixed, just not sure about the missing manifest message.

**Pull request merged**